docs/client_reference.rst

```diff
@@ -1168,17 +1170,19 @@ manually.
       :raise TypeError: if value returned by ``dumps(data)`` is not
                         :class:`str`
 
-   .. comethod:: close(*, code=1000, message=b'')
+   .. method:: close(*, code=1000, message=b'')
 
-      A :ref:`coroutine<coroutine>` that initiates closing handshake by sending
+      A method that initiates closing handshake by sending
       :const:`~aiohttp.WSMsgType.CLOSE` message. It waits for
-      close response from server. It add timeout to `close()` call just wrap
-      call with `asyncio.wait()` or `asyncio.wait_for()`.
+      close response from server. To add a timeout to `close()` call
+      just wrap the call with `asyncio.wait()` or `asyncio.wait_for()`.
```

Comment:
Non-coroutine cen not be waited. So text about timeout and waiting is completely wrong. Does not it ?